docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -0,0 +1,78 @@
+# Upgrade to 8.0.0
+
+## New dependencies
+
+Depending on the used OS, run the corresponding command.
+
+### FreeBSD
```

Comment:
I suggest that we preserve the order CentOS : Debian/Ubuntu : FreeBSD in all places.